src/js/app.js

```diff
@@ -32,6 +32,12 @@ hljs.registerLanguage('sql', sql);
 import erb from 'highlight.js/lib/languages/erb';
 hljs.registerLanguage('erb', erb);
 
+import xml from 'highlight.js/lib/languages/xml';
+hljs.registerLanguage('xml', xml);
```

Comment:
I don't see any xml code blocks in the guides. We can probably skip this, right?

Reply:
I included it because I wasn't sure how the plugin handled it because it seems to include HTML and XML together in the same file. I think we should be safe removing it.